package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "ng serve",
     "build": "ng build --target=production --aot",
-    "coverage": "ng test --no-watch --code-coverage --single-run=true",
+    "coverage": "ng test --environment=dev --no-watch --code-coverage --single-run=true --sourcemaps=false",
```

Comment:
Oh sorry, I was unclear in the question. Can you tell me why this change is necessary? Do the unit tests not work with the default flags?

I'm open to them if they're necessary, but I don't want to add extra flags if we don't need to.